### PR TITLE
integrate homepage animation

### PR DESCRIPTION
**Pull request by davist11**
* Using Elliott's animation: https://lottiefiles.com/share/OC06Sf
* Text is coming from the CMS
* TODO: all the responsiveness (and the rest of the homepage)

### CODE REVIEW

**Comment by jeremyfrank** [comment]
Just a note, I was seeing an exclamation icon when previewing the build on Netlify, but now it's working. Not sure what happened.

image

File: src/components/HomepageAnimation.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+                {
+                    visibility: [0, 0.1],
+                    type: 'stop',
+                    frames: [0],
+                },
+                {
+                    visibility: [0.1, 1],
+                    type: 'seek',
+                    frames: [0, 870],
+                },
```
[the remark this answers is not in the document]
Comment (davist11) [reply src/components/HomepageAnimation.js:49]: yeah constants make sense; I think it's unnecessary to move these to the CMS

**Comment by davist11** [comment]
> It would be cool if when you stop scrolling, the animation could complete the currently playing piece, that way the animation doesn't get stuck in a halfway done state

Hmm confused that you mean. The animation progress is directly tied to scrolling

**Comment by jeremyfrank** [comment]
> Hmm confused that you mean. The animation progress is directly tied to scrolling

In my screenshot, that's where I stopped scrolling. The red triangle was partially drawn. It would be cool if each individual triangle animation could start at a certain scroll position and complete even if you stop scrolling, kind of like triggering a css animation; once it starts it plays through to the end. Not sure what's possible with Lottie output.

**Pull request merged**